src/components/lifetime/PriceBannerForCampaigns.tsx

```diff
@@ -9,18 +10,28 @@ export const PriceBannerForCampaigns = ({ textContent }) => {
   const router = useRouter();
 
   function handleOnClick() {
-    router.push('/lifetime');
+    window.location.hash = 'billingButtons';
```

Comment:
It is preferred to use `href="#billingButtons"` in the HTML element than using JavaScript window.location.hash to scroll to that element.